gitpod-io/gitpod
# Don't deploy Jaeger with preview environments

## Description

With the adoption of honeycomb not only for production but for our preview environments as well, we're finally getting rid of anything related to Jaeger inside our preview environments.

## How to test

Open a workspace from this PR, then:
* Notice that jaeger is not deployed with the preview. 


## Release Notes

```release-note
NONE
```

## Follow up

The Jaeger CRD is still present in the cluster, and also the Jaeger-Operator inside the `monitoring` namespace.
I believe we need to get rid of both of them to say that https://github.com/gitpod-io/ops/issues/831 is done
